src/jit/gentree.h

```diff
@@ -1080,6 +1080,17 @@ struct GenTree
     bool OperIsInitBlkOp() const;
     bool OperIsDynBlkOp();
 
+    static
+    bool OperIsBlk(genTreeOps gtOper)
+    {
+        return (gtOper == GT_OBJ);
+    }
+
+    bool OperIsBlk()
+    {
```

Comment:
(should it be GT_BLK?)

---
In reply to: [75344942](https://github.com/dotnet/coreclr/pull/6780#discussion_r75344942) [](ancestors = 75344942,75344836)